### ISSUE TITLE
FxA-78: node4 upgrade

### ISSUE THREAD

**Issue by rfk** [mozilla/fxa-features#19]

**Comment by rfk** [comment]
plan is to ship this between train-66 and train-67

**Comment by rfk** [comment]
new plan is to ship this between train-67 and train-68

**Comment by rfk** [comment]
We tried shipping this with train-67, had to rollback.  We think we've fixed the issue and will try again soon.

**Comment by davismtl** [comment]
Auth servers landed but we still need to land oauth and profile servers. We need to sync up with jbuck and jon.

**Comment by davismtl** [comment]
from mtg
will finish this after next train

**Comment by davismtl** [comment]
from mtg:
In final stages. Almost ready to unblock the reminders.
Not sure of which train yet. (hoping next 2 trains will be done with it)
